controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -458,13 +505,13 @@ def test_map_volume_no_availabe_lun(self):
 
         self.mediator.client.cmd.mapping_list.return_value = Mock(as_list=[mapping1, mapping2, mapping3])
         with self.assertRaises(array_errors.NoAvailableLunError):
-            self.mediator.map_volume("vol", "host")
+            self.mediator.map_volume("vol-wwn", "host")
```

Comment:
if you already here, `volume-wwn`?

Reply:
this change is a small alignment to match most existing values, but your request is relevant to **all** existing values, and this PR diff is already large enough as it is.
I hope to make a separate renaming PR, if so, I can include it there

Reply:
great